high_card_rb.rb

```diff
@@ -0,0 +1,59 @@
+require "pry"
+
+# Use these two arrays to generate a deck of cards.
+Ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
+Suits = [ "hearts", "spades", "clubs", "diamonds" ]
```

Comment:
The only time we capitalize the first letter of a variable in Ruby is when defining a class (e.g., `class Person`).